### DISCUSSION TITLE
Add Seed Cluster for CE - Minio problem

### DISCUSSION THREAD

**Discussion by clickersmudge** [kubermatic/kubermatic#11297] [Q&A] [answered]
Hi,
We are trying to add a calster in "Large-Scale Deployments" architecture.

We deploy new cluster with KubeOne.
Login to this cluster and deploy as seed.
```
export KUBECONFIG=/path/to/seed-cluster/kubeconfig
./kubermatic-installer deploy kubermatic-seed --config kubermatic.yaml --helm-values values.yaml
```

Unfortunately, the installation does not pass through Minio
```
./kubermatic-installer deploy kubermatic-seed --config kubermatic.yaml --helm-values values-seed.yaml --skip-dependencies
INFO[0000] 🚀 Initializing installer…                     edition="Community Edition" version=v2.21.2
INFO[0000] 🚦 Validating the provided configuration…     
INFO[0000] ✅ Provided configuration is valid.           
INFO[0000] 🚦 Validating existing installation…          
INFO[0000] ✅ Existing installation is valid.            
INFO[0000] 🛫 Deploying KKP seed stack…                  
INFO[0000]    💾 Deploying kubermatic-fast StorageClass… 
INFO[0001]    ✅ StorageClass exists, nothing to do.     
INFO[0001]    📦 Deploying Minio…                        
ERRO[0308] ❌ Operation failed: failed to deploy Minio: failed to deploy Helm release: failed to install: Error: release minio failed, and has been uninstalled due to atomic being set: timed out waiting for the condition. 
```

Is it possible to add seed cluster without deploying minio?

**Comment by embik** [comment, marked as answer]
Hi @clickersmudge,

it's not possible to say what exactly is happening. I would recommend to observe the pods on your new Seed cluster and run `kubectl describe` and `kubectl logs` on the Minio pods that do not seem to come online. From the error message it is only clear that the Helm chart is failing to install because it times out, which usually means that pods fail to start. But the reason for that is opaque to us right now.

Skipping the Minio installation is not possible as far as I know.

That being said, judging by the other discussions, you already have a working master/seed (possibly a shared) installation. Please be aware that the Community Edition of KKP only allows a singleton seed called `kubermatic` and that adding additional seeds to your master requires the Enterprise Edition. Even if you succeed with the Minio installation, this might block you from adding the new environment to your KKP setup.

**Reply by clickersmudge** [reply]
@embik Thanks for time. We'll check the tip with `kubectl describe` during installation. We have two variant for testing. One master/seed shared installation (working). And we wanted to check another one scenario in the  "Large-Scale Deployments (different KubeOne KKP, installation). I understand that a one seed called "kubermatic" can work in this scenario. The next ones require a Enterprise version?

**Reply by embik** [reply]
Okay, yes, if this is another KKP installation it should not be a problem. As you said, the Community Edition is capable of managing a single seed (either "shared" / co-located with the master or on a dedicated, second cluster), additional seeds under the same KKP installation require the Enterprise Edition.
